tfhe/src/error.rs

```diff
@@ -65,3 +65,12 @@ impl From<std::convert::Infallible> for Error {
         unreachable!()
     }
 }
+
+/// Error returned when the provided range for a slice is invalid
+#[derive(Debug)]
+pub enum InvalidRangeError {
```

Comment:
Since this is meant to be an error, I think it should impl std::error::Error, and maybe also Into<tfhe::Error> to allow  using `?`

Reply:
done